crates/daphne/src/protocol/aggregator.rs

```diff
@@ -412,6 +412,7 @@ impl DapTaskConfig {
         part_batch_sel: &PartialBatchSelector,
         agg_param: &DapAggregationParam,
         reports: S,
+        replay_protection: ReplayProtection,
```

Comment:
I thought about adding it to task config but I decide not to because I I'm wary of overloading the role of the task config. This is supposed to be more of an ad-hoc parameter passed in tests.

On the point of production, this is used by the leader only, which is not in production, but yes, it could be used in a production leader but I expect the code will change again by then. Either way changing the parameter name will not change anything in the call site so I don't think it improves readability. We could make this parameter available in tests only though :thinking:

Reply:
fenced behind the feature flag. Maybe the feature flag itself should be "insecure-test-utils" :thinking: